skxray/fitting/base/parameter_data.py

```diff
@@ -136,4 +137,189 @@ def get_para():
        based on different algorithms.
        Use copy for dict.
     """
-    return para_dict
\ No newline at end of file
+    return para_dict
+
+default_param = {
+    "coherent_sct_amplitude": {
+        "adjust_element": "none",
+        "bound_type": "none",
+        "e_calibration": "none",
+        "fit_with_tail": "none",
+        "free_more": "none",
+        "linear": "none",
+        "max": 10000000.0,
+        "min": 1.0,
+        "value": 100000
```

Comment:
It would be nice to have a short description of where these values come from with a link, if it exists.